python/ray/serve/_private/controller.py

```diff
@@ -511,8 +511,8 @@ def shutdown(self):
 
         if self._shutdown_start_time is None:
             self._shutdown_start_time = time.time()
+            logger.info("Controller shutdown started!", extra={"log_to_stderr": False})
```

Comment:
Remove the exclamation point please (we should basically never have these in logs)